spec/facades/cocktail_facade_spec.rb

```diff
@@ -47,6 +47,41 @@
     end
   end
 
+  describe '::retrieve_details' do
+    before :each do
+      User.destroy_all
+      Cocktail.destroy_all
+      Rating.destroy_all
+    end
+
+    it 'returns json with cocktail details and rating' do
+      user = create(:user)
+
+      response_body = File.read('./spec/fixtures/random_cocktail.json')
+        stub_request(:get, "https://www.thecocktaildb.com/api/json/v1/1/random.php?api_key=#{ENV['cocktail_key']}")
```

Comment:
Keep consistent and use vcr?

Reply:
@markcyen actually, it is using vcr still. The randomness caused some issues, but it's actually using vcr due to the call to the gem in top line of the file (`RSpec.describe` line)